src/components/SectionHeader.tsx

```diff
@@ -10,21 +10,20 @@ type Props = {
 };
 
 export const SectionHeader = ({ title, onPress }: Props) => {
-  const innerComponent = <Title>{title}</Title>;
+  const innerComponent = (
+    <Title
+      accessibilityLabel={`(${title}) ${consts.a11yLabel.heading} ${
+        onPress ? consts.a11yLabel.button : null
+      } `}
+    >
+      {title}
+    </Title>
+  );
 
```

Comment:
**Is this reaching all titles across the app ?**  ( see code above )

If yes , I don't actually need a11y labels for titles in the specific screens like this one and many others? ( see code below ) 
<img width="640" alt="Screenshot 2021-07-12 at 13 25 11" src="https://user-images.githubusercontent.com/43641321/125279897-c0e4c000-e314-11eb-95ed-5735263ab1a4.png">

Reply:
I do not see how the code from the screenshot correlates to the section header. so i don't know what you are trying to ask here.

Reply:
I am still confused about the correlation here.
In which way is the SectionHeader involved here?

Reply:
Well I suspected that title component in `SectionHeader` and title in `ServiceScreen` are doing the same job . But I guess from your answer that I misunderstood what is  `SectionHeader` functionality, I thought it handle all titles across the app. But what does it do instead ?